client/my-sites/domains/domain-management/components/point-to-wpcom-dialog.tsx

```diff
@@ -0,0 +1,42 @@
+import { useI18n } from '@wordpress/react-i18n';
+import AcceptDialog from 'calypso/lib/accept/dialog'; // eslint-disable-line no-restricted-imports
+
+type PointToWpcomDialogProps = {
+	visible: boolean;
+	onClose: ( accepted: boolean ) => void;
+};
+
+export const PointToWpcomDialog = ( { visible, onClose }: PointToWpcomDialogProps ) => {
+	const { __ } = useI18n();
```

Comment:
```suggestion
	if ( ! visible ) {
		return null;
	}

	const { __ } = useI18n();
```
Suggestion: what do you think of adding an early return to avoid the ternary at the end of the component?